packages/daimo-api/src/network/bundlerClient.ts

```diff
@@ -243,6 +175,23 @@ function packGasFees({
 function packHiLo(hi: bigint, lo: bigint) {
   return concatHex([
     numberToHex(hi, { size: 16 }),
-    numberToHex(lo, { size: 32 }),
+    numberToHex(lo, { size: 16 }),
```

Comment:
what is this change for?

Reply:
fixing a bug. the correct serialization for the EntryPoint v0.7 PackedUserOp = in a few spots, needs to pack two uint128s into a single bytes32